File: javascript/employee.cs
class Employee
{
    string Name;
    string Design;
    Date Dob;

    Employee(string name, string design, Date dob)
    {
        this.Name=name;
        this.Design=design;
        this.Dob=dob;

    }
     
    void CalcAge()
    {
        console.log("age");
    }
}

class Main
{
    void main()
    {
        Employee e1=new Employee("abc",abc,2000);
        Employee e2=new Employee("mno",mno,2000);
        Employee e3=new Employee("xyz",xyz,2000);
    }
}

File: javascript/employee.js
function Employee(name,design,year)
{
    this.name=name;
    this.design=design;
    this.year=year;
}

Employee.prototype.calcAge=function()
{
    console.log("age of "+this.name+" is: "+(2021-this.year));
}

console.log(Employee.prototype);

let emp1=new Employee("Alex","tester", 1995);
console.log(emp1);
emp1.calcAge();